src/tests.cpp

```diff
@@ -1,8 +1,16 @@
 // SPDX-FileCopyrightText: 2024-2025 Filipe Coelho <falktx@darkglass.com>
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+#include "config.h"
```

Comment:
all these incldues are not needed and not wanted, likely an issue from your IDE. please remove them.